rpc-jobs/jjb-setup.yml

```diff
@@ -1,4 +1,24 @@
 - job:
+    name: 'Merge-Trigger-JJB'
+    project-type: freestyle
```

Comment:
This job wasnt working when I left work, so worth testing it with Matt's typo fix before basing anything else on this.